### PR TITLE
Add content from Lost Omens Firebrands

### PR DESCRIPTION
**Pull request by TikaelSol**
Also credit to Dogstarrb for this one.

### CODE REVIEW

File: packs/data/equipment.db/divers-gloves-moderate.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    "action:swim"
+                ],
+                "selector": "athletics",
+                "type": "item",
+                "value": 2
+            },
+            {
+                "key": "BaseSpeed",
+                "selector": "swim",
+                "value": "@actor.attributes.speed.total*0.5"
```
Comment (simonward) [inline packs/data/equipment.db/divers-gloves-moderate.json:51]: floor

File: packs/data/deities.db/arshea.json [hunk 2]
```diff
@@ -17,10 +17,14 @@
...
...on</p>\n<p><strong>Areas of Concern</strong> freedom, physical beauty, and sexuality</p>\n<h2>Avatar</h2>\n<p>When casting the @UUID[Compendium.pf2e.spells-srd.Avatar]{Avatar} spell, a worshipper of Arshea becomes a fluid, shifting humanoid that changes appearance, gender, and form moment by moment, though the caster can choose one specific form if desired. Dressed in gossamer sashes, scarves, and veils, each form wields a flail and possesses magnificent, wispy wings of gray and silver feathers. They gain the following additional abilities.</p>\n<p><strong>Arshea</strong> Speed 30 feet, fly 70 feet, immune to forced movement, immune to @UUID[Compendium.pf2e.conditionitems.Immobilized]{Immobilized}</p>\n<p>Melee <span class=\"\\&quot;pf2-icon\\&quot; pf2-icon\">1</span> flail (disarm, reach 15 feet, sweep, trip), Damage [[/r (6d6+6)[bludgeoning]]]</p>\n<p>Melee <span class=\"\\&quot;pf2-icon\\&quot; pf2-icon\">1</span> gossamer veil (disarm, finesse, grapple, nonlethal, reach 20 feet, trip), Damage [[/r (6d4+6)[slashing]]]</p>\n<p>Ranged <span class=\"\\&quot;pf2-icon\\&quot; pf2-icon\">1</span> gaze of abandon (nonlethal, range 120 feet, visual), Damage [[/r (6d6+3)[mental]]].</p>"
```
Comment (simonward) [inline packs/data/deities.db/arshea.json:20]: Cruft

File: packs/data/feats.db/rolling-landing.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                }
+            ]
+        },
+        "rules": [],
+        "source": {
+            "value": "Pathfinder Lost Omens: Firebrands"
+        },
+        "traits": {
+            "rarity": "common",
+            "value": []
```
Comment (simonward) [inline packs/data/feats.db/rolling-landing.json:35]: Is this missing traits?

File: packs/data/feats.db/it-was-me-all-along.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            "max": 1,
+            "per": "day"
+        },
+        "level": {
+            "value": 14
+        },
+        "prerequisites": {
+            "value": [
+                {
+                    "value": "Alter Ego Dedication or,Vigilante Dedication"
```
Comment (simonward) [inline packs/data/feats.db/it-was-me-all-along.json:26]: comma

File: packs/data/feats.db/rolling-landing.json (hunk 3 above)
Comment (TikaelSol) [reply packs/data/feats.db/rolling-landing.json:35]: nope, general and skill are set automatically by the system.

**Review by SpartanCPA — APPROVED** [review]

**Review by simonward — APPROVED** [review]

**Pull request merged**